app/views/course/assessment/submission/submissions/_questions.json.jbuilder

```diff
@@ -7,8 +7,17 @@ topic_ids_hash = submission.submission_questions.map do |sq|
 end.to_h
 
 json.questions assessment.questions do |question|
-  json.partial! 'question', question: question, can_grade: can_grade
+	answerId = answer_ids_hash[question.id]
+	answer = answers.select{|ans| ans.id == answerId}.first.specific
```

Comment:
😄 space before and after `{`

Also use `detect` instead of `select {}.first`. What if answer is `nil` ?